dummyserver/server.py

```diff
@@ -16,6 +16,11 @@
 import warnings
 from datetime import datetime
 
+try:
+    from ssl import CERT_OPTIONAL
+except:
+    CERT_OPTIONAL = None
```

Comment:
Cool, let's do that then and let the CI catch any issues. Assuming there aren't any, my note about skipping tests also naturally becomes irrelevant. :smile: